test/test_allocation_domain.cpp

```diff
@@ -1121,10 +1121,12 @@ TEST_F(NVFuserTest, AllocationDomainContiguityIssue1021) {
   Fusion* fusion = fusion_ptr.get();
   FusionGuard fg(fusion);
 
-  auto tv0 = TensorViewBuilder().ndims(2).shape({-1, -1}).build();
-
-  std::vector<IterDomain*> alloc_domain = {tv0->axis(1), tv0->axis(0)};
-  tv0->setAllocationDomain(alloc_domain, {false, true});
+  auto tv0 = TensorViewBuilder()
+                 .ndims(2)
+                 .shape({-1, -1})
+                 .contiguity({false, true})
+                 .strideOrder({0, 1})
```

Comment:
Wait, stride_order is minor-to-major? Otherwise, this line changes the behavior.

Reply:
So if we look at the alloc_domain above, the old code has:
`std::vector<IterDomain*> alloc_domain = {tv0->axis(1), tv0->axis(0)};`

Which is basically a transposed matrix, and it maps to `strideOrder({0, 1})` here, since the semantics for stride order means that dimension 0 is the inner dimension.